config/routes.rb

```diff
@@ -3,4 +3,14 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+ 
+  get 'dishes/:id', to: 'dishes#show', as: 'dish'
+  post 'dishes/:id/add_ingredient', to: 'dishes#add_ingredient', as: 'add_ingredient_to_dish'
```

Comment:
Should be `post 'dishes/:id/ingredients', to: 'ingredients#create', as: 'add_ingredient_to_dish'` OR `post 'dishes/:id/ingredients', to: 'dish_ingredients#create', as: 'add_ingredient_to_dish'`